### ISSUE TITLE
How to run the demo?

### ISSUE THREAD

**Issue by onmyway133** [hyperoslo/Flamingo#35]
Hi 😄 

I see that I can get the demo to work by
- Run `swift build`
- Run `.build/debug/Demo`
- Go to `localhost:8080`

Is it the correct way? Should we add the instructions to README?

**Comment by vadymmarkov** [comment]
@onmyway133 Yes, we should add this to README. But the idea is to make a separate repo with only a demo, so I think it's the next step.
